pkg/littlefs/Makefile

```
@@ -8,10 +8,8 @@ include $(RIOTBASE)/pkg/pkg.mk
 
 CFLAGS += -Wno-format
 
-# Disable debug printing
-ifneq ($(DEVELHELP),1)
-  CFLAGS += -DLFS_NO_DEBUG -DLFS_NO_WARN -DLFS_NO_ERROR
-endif
+# replace pkg supplied logging macros with RIOT's
+CFLAGS += -include $(PKG_DIR)/lfs_log.h
```

```suggestion
CFLAGS += -I$(PKG_DIR)/lfs_log.h
```

That's how I've seen it mostly in RIOT, but I guess that's a matter of taste.

---

see [other comment](https://github.com/RIOT-OS/RIOT/pull/20894#discussion_r1792214824) about this.